### ISSUE TITLE
Carthage build for iOS fails with exit code 65

### ISSUE THREAD

**Issue by sherburne** [Nike-Inc/SQift#27]
Hello! Thanks for putting this library together; it's pretty cool.

While building the latest (5.0.0) SQift via Carthage for iOS using `carthage build SQift --platform ios`, I ran into this error:

```
*** Building scheme "SQift iOS" in SQift.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/SQift.xcodeproj -scheme SQift\ iOS -configuration Release -derivedDataPath /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=27CDD446-9FCE-49FB-8D40-25354D030AC1 -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift)
```

Here's the relevant bit of Carthage logging:

```
CompileSwift normal i386 (in target: SQift iOS Tests)
    cd /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/FunctionTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Errors/ExpressibleByRowErrorTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Helpers/TestTables.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Extensions/StringExtensionTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/BusyTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/BackupTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Errors/SQLiteErrorTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Rows/RowTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/AuthorizerTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/TraceTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/ConnectionTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/TransactionTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/CheckpointTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/ConnectionQueueTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Database/TableLockPolicyTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Database/MigratorTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/StatementTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/BaseTestCase.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/QueryTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Database/DatabaseTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Extensions/ProcessInfoExtension.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Extensions/FileManagerExtension.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/ConnectionPoolTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/CollationTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/HookTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Rows/BindingTests.swift /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Extensions/DispatchQueueExtension.swift -supplementary-output-file-map /var/folders/yx/mj_vb0ns2dl2qszddbdftljnvpk5bd/T/supplementaryOutputs-759a89 -target i386-apple-ios10.0-simulator -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -I /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Products/Release-iphonesimulator -F /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Products/Release-iphonesimulator -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks -g -module-cache-path /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/ModuleCache.noindex -profile-generate -profile-coverage-mapping -swift-version 5 -enforce-exclusivity=checked -O -serialize-debugging-options -serialize-debugging-options -Xcc -working-directory -Xcc /opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift -enable-anonymous-context-mangled-names -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/SQift\ Tests-generated-files.hmap -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/SQift\ Tests-own-target-headers.hmap -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/SQift\ Tests-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/SQift\ Tests-project-headers.hmap -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Products/Release-iphonesimulator/include -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/DerivedSources-normal/i386 -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/DerivedSources/i386 -Xcc -I/Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/DerivedSources -module-name SQift_Tests -num-threads 8 -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/FunctionTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/ExpressibleByRowErrorTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/TestTables.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/StringExtensionTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/BusyTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/BackupTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/SQLiteErrorTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/RowTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/AuthorizerTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/TraceTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/ConnectionTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/TransactionTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/CheckpointTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/ConnectionQueueTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/TableLockPolicyTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/MigratorTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/StatementTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/BaseTestCase.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/QueryTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/DatabaseTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/ProcessInfoExtension.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/FileManagerExtension.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/ConnectionPoolTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/CollationTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/HookTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/BindingTests.o -o /Users/tsherburne/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/SQift/5.0.0/Build/Intermediates.noindex/SQift.build/Release-iphonesimulator/SQift\ iOS\ Tests.build/Objects-normal/i386/DispatchQueueExtension.o
/opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/Functions/TraceTests.swift:12:18: error: module 'SQift' was not compiled for testing
@testable import SQift
                 ^
/opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/ConnectionTests.swift:12:18: error: module 'SQift' was not compiled for testing
@testable import SQift
                 ^
/opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Database/MigratorTests.swift:12:18: error: module 'SQift' was not compiled for testing
@testable import SQift
                 ^
/opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/StatementTests.swift:12:18: error: module 'SQift' was not compiled for testing
@testable import SQift
                 ^
/opt/test/Projects/test-git/platforms/ios/Carthage/Checkouts/SQift/Tests/Tests/Connection/ConnectionPoolTests.swift:12:18: error: module 'SQift' was not compiled for testing
@testable import SQift

```

I was able to get the build working following this bit of advice about setting the `Enable Testability` flag for the build: https://stackoverflow.com/questions/30787674/module-was-not-compiled-for-testing-when-using-testable

Let me know if you need additional info.

**Comment by cnoon** [comment]
Ah yeah this is due to the test suite being attached to the framework when compiled. We can get this fixed up @sherburne, thanks for reporting! 🍻

**Comment by cnoon** [comment]
This issue has been resolved in #28 and will be released shortly as SQift 5.0.1. Thanks again @sherburne for reporting. 🍻
